cmake/Platform/Linux/Packaging_linux.cmake

```diff
@@ -69,4 +69,24 @@ elseif("$ENV{O3DE_PACKAGE_TYPE}" STREQUAL "DEB")
         ${CMAKE_BINARY_DIR}/cmake/Platform/Linux/Packaging/postrm
     )
 
+    # Scripts to add executable files to PATH
+    configure_file("${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.in"
+        ${CMAKE_BINARY_DIR}/Packaging/o3de
+        @ONLY
+    )
+    configure_file("${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.assetprocessor.in"
+        ${CMAKE_BINARY_DIR}/Packaging/o3de.assetprocessor
+        @ONLY
+    )
+    configure_file("${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.editor.in"
+        ${CMAKE_BINARY_DIR}/Packaging/o3de.editor
+        @ONLY
+    )
+    ly_install_files(PROGRAMS FILES
+        ${CMAKE_BINARY_DIR}/Packaging/o3de
+        ${CMAKE_BINARY_DIR}/Packaging/o3de.assetprocessor
+        ${CMAKE_BINARY_DIR}/Packaging/o3de.editor
+        DESTINATION /usr/bin
```

Comment:
Not to throw too much of a wrench here, but why not use `/usr/local/bin`? Generally this should be in `PATH` and helps folks know that it was installed outside of the standard package manager process.

Reply:
Resolved in https://github.com/o3de/o3de/pull/16443/commits/6efe64037df656eb25c36d0ada9231f338370756